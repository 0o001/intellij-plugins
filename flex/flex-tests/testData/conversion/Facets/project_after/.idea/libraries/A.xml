<?xml version="1.0" encoding="UTF-8"?>
<component name="libraryTable">
  <library name="A">
    <CLASSES>
      <root url="jar://$MODULE_DIR$/../lib/A.swc!/" />
    </CLASSES>
    <JAVADOC />
    <SOURCES />
  </library>
</component>

