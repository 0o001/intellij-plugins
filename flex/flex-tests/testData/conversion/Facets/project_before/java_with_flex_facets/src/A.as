/**
 * Created by IntelliJ IDEA.
 * User: Kirill.Safonov
 * Date: 12.10.11
 * Time: 18:32
 * To change this template use File | Settings | File Templates.
 */
package {
import flash.display.Sprite;

public class A extends Sprite {
    public function A() {
    }
}
}
