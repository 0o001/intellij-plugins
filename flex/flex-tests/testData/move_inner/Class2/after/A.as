package {
public class A {
  protected var i : int;
}
}

