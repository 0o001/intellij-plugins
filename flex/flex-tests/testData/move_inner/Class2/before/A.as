package {
public class A {
  protected var i : int;
}
}

class B<caret> extends A {
  public function foo() {
    i++;
  }
}