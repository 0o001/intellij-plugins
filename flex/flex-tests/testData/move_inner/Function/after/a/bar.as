package a {
/**
 * this is foo docs
 */
internal function bar() {
    var a:From;
}
}
