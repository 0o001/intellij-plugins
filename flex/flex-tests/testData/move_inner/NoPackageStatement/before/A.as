function m<caret>y():void{}
