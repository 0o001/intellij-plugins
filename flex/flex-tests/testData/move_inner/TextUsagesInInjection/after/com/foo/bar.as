package com.foo {
function bar() {
}
}
