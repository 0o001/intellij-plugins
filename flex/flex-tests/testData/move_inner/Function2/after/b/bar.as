package b {
import a.From;

/**
 * this is foo docs
 */
public function bar() {
    var a:From;
}
}
