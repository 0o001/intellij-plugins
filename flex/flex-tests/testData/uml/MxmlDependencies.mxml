<caret expected=""><?xml version="1.0"?>
<s:Application xmlns:fx="http://ns.adobe.com/mxml/2009" xmlns:s="library://ns.adobe.com/flex/spark"
               xmlns:mx="library://ns.adobe.com/flex/mx" xmlns:mxComps="*">
  <s:Button label="foo" skinClass="MxmlDependencies_4"/>
  <s:Group id="mygroup"/>
  <fx:Script><![CDATA[
    import com.bar.A;

    var v = new Button();

    var v2 : Foo;
    var v3 : Vector.<Bar>;
    ]]></fx:Script>

  <fx:Script>
    function aa() {
      new Hello();
    }
  </fx:Script>
  <mxComps:MxmlDependencies_3 label="from MXML" id="myComponent"/>
  <s:List itemRenderer="com.foo.MyRenderer"/>
  <fx:Style>
    @namespace mx "library://ns.adobe.com/flex/mx";
    mx|Button {
      upSkin: ClassReference("MySkin");
    }  </fx:Style>
</s:Application>