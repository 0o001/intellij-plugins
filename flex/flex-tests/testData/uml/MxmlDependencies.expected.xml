<?xml version="1.0" encoding="UTF-8"?>
<umlModel origin="MxmlDependencies">
  <node id="Bar" nodeName="Class Bar" name="&lt;html&gt;&lt;b&gt;Bar&lt;/b&gt;&lt;/html&gt;" />
  <node id="Foo" nodeName="Class Foo" name="&lt;html&gt;&lt;b&gt;Foo&lt;/b&gt;&lt;/html&gt;" />
  <node id="Hello" nodeName="Class Hello" name="&lt;html&gt;&lt;b&gt;Hello&lt;/b&gt;&lt;/html&gt;" />
  <node id="MxmlDependencies" nodeName="Class MxmlDependencies" name="&lt;html&gt;&lt;b&gt;MxmlDependencies&lt;/b&gt;&lt;/html&gt;">
    <element name="aa()" type="*" category="Methods" visibility="PACKAGE_LOCAL" />
    <element name="v" type="*" category="Fields" visibility="PACKAGE_LOCAL" />
    <element name="v2" type="Foo" category="Fields" visibility="PACKAGE_LOCAL" />
    <element name="v3" type="Vector.&lt;Bar&gt;" category="Fields" visibility="PACKAGE_LOCAL" />
  </node>
  <node id="MxmlDependencies_3" nodeName="Class MxmlDependencies_3" name="&lt;html&gt;&lt;b&gt;MxmlDependencies_3&lt;/b&gt;&lt;/html&gt;" />
  <node id="MxmlDependencies_4" nodeName="Class MxmlDependencies_4" name="&lt;html&gt;&lt;b&gt;MxmlDependencies_4&lt;/b&gt;&lt;/html&gt;">
    <element name="hostComponent" type="Panel" visibility="PACKAGE_LOCAL" />
  </node>
  <node id="MySkin" nodeName="Class MySkin" name="&lt;html&gt;&lt;b&gt;MySkin&lt;/b&gt;&lt;/html&gt;" />
  <node id="com.bar.A" nodeName="Class com.bar.A" name="&lt;html&gt;&lt;b&gt;com.bar.A&lt;/b&gt;&lt;/html&gt;" />
  <node id="com.foo.MyRenderer" nodeName="Class com.foo.MyRenderer" name="&lt;html&gt;&lt;b&gt;com.foo.MyRenderer&lt;/b&gt;&lt;/html&gt;" />
  <node id="spark.components.Button" nodeName="Class spark.components.Button" name="&lt;html&gt;&lt;b&gt;spark.components.Button&lt;/b&gt;&lt;/html&gt;">
    <element name="Button()" category="Constructors" visibility="PUBLIC" />
    <element name="_emphasized" type="Boolean" category="Fields" visibility="PRIVATE" />
    <element name="emphasized" type="Boolean" category="Properties" visibility="PUBLIC" />
    <element name="styleName" type="Object" category="Properties" visibility="PUBLIC" />
    <element name="emphasizeStyleName()" type="void" category="Methods" visibility="PRIVATE" />
    <element name="skinParts" type="Object" category="Properties" visibility="PROTECTED" />
    <element name="VERSION" type="String" category="Fields" visibility="PACKAGE_LOCAL" />
    <element name="_skinParts" type="Object" category="Fields" visibility="PRIVATE" />
    <element name="mx_internal" type="*" category="Fields" visibility="PRIVATE" />
  </node>
  <edge from="MxmlDependencies" to="Bar" idElement="MxmlDependencies" relship="ONE_TO_MANY" relshipLabel="v3" />
  <edge from="MxmlDependencies" to="Foo" idElement="MxmlDependencies" relship="ONE_TO_ONE" relshipLabel="v2" />
  <edge from="MxmlDependencies" to="Hello" idElement="MxmlDependencies" relship="CREATE" relshipLabel="&lt;html&gt;&amp;laquo;create&amp;raquo;&lt;/html&gt;" />
  <edge from="MxmlDependencies" to="MxmlDependencies_3" idElement="MxmlDependencies" relship="ONE_TO_ONE" relshipLabel="myComponent" />
  <edge from="MxmlDependencies" to="MxmlDependencies_4" idElement="MxmlDependencies" relship="DEPENDENCY" relshipLabel="skinClass" />
  <edge from="MxmlDependencies" to="MySkin" idElement="MxmlDependencies" relship="DEPENDENCY" relshipLabel="upSkin" />
  <edge from="MxmlDependencies" to="com.foo.MyRenderer" idElement="MxmlDependencies" relship="DEPENDENCY" relshipLabel="itemRenderer" />
  <edge from="MxmlDependencies" to="spark.components.Button" idElement="MxmlDependencies" relship="CREATE" relshipLabel="&lt;html&gt;&amp;laquo;create&amp;raquo;&lt;/html&gt;" />
  <edge from="MxmlDependencies" to="spark.components.Button" idElement="MxmlDependencies" relship="DEPENDENCY" />
</umlModel>

